src/nspawn/nspawn.c

```diff
@@ -2170,6 +2222,7 @@ static int copy_devnodes(const char *dest) {
                 "random\0"
                 "urandom\0"
                 "tty\0"
+                "fuse\0"
```

Comment:
Make this list a strv, and then dynamically add "fuse" if support is detected, rather than skipping inline while processing